qubes/tests/integ/vm_qrexec_gui.py

```diff
@@ -909,8 +909,14 @@ def test_000_anyvm_deny_dom0(self):
             ' qrexec-client-vm output: {} {}'.format(stdout, stderr))
 
 def create_testcases_for_templates():
-    return qubes.tests.create_testcases_for_templates('TC_00_AppVM',
-        TC_00_AppVMMixin, qubes.tests.SystemTestCase,
+    yield from qubes.tests.create_testcases_for_templates('TC_00_AppVM',
```

Comment:
The new class prefix must be different for each class too - usually I match them with the actual base class name